pygame_gui/core/utility.py

```diff
@@ -468,10 +468,13 @@ def load(self):
                         self.force_style,
                         self.style,
                     )
-            except (pygame.error, OSError):
-                error = FileNotFoundError(
-                    f"Unable to load resource with path: {str(self.location)}"
-                )
+            except (pygame.error, OSError) as exc:
+                if (not isinstance(exc, pygame.error)):
+                    error = FileNotFoundError(
+                        f"Unable to load resource with path: {str(self.location)}"
+                    )
```

Comment:
I'm not convinced this is ideal either (I mean, of course, reaching ideals is hard anyway), but suppose you get a permission error, which is an os error as well, that information will once again be lost.